code/game/objects/items/weapons/tanks/jetpack.dm

```diff
@@ -27,7 +27,6 @@
 	. = ..()
 	if(air_contents.total_moles < 5)
 		user << "<span class='danger'>The meter on \the [src] indicates you are almost out of gas!</span>"
-		playsound(user, 'sound/effects/alert.ogg', 50, 1)

```

Comment:
Agreed, lazy solution.

Also, better method would be to time lock the alarm. Basically, every time someone examines, check if it's been some x minutes since alarm last played.